addons/medical_treatment/functions/fnc_healTime.sqf

```diff
@@ -18,13 +18,14 @@
 params ["_unit"];
 
 private _totalDamage = 0;
+private _treattime = 0;
 
 {
     _totalDamage = _totalDamage + _x;
 } forEach (_unit getVariable [QEGVAR(medical,bodyPartDamage), []]);
-if (QEGVAR(medical,PAKTime) != 0) then {
-    _time = QEGVAR(medical,PAKTime);
+if (EGVAR(medical,PAKTime) > 0) then {
+    _treattime = EGVAR(medical,PAKTime);
```

Comment:
`_treattime` should probably be `_treatTime`. Camel case or whatever it's called.